### ISSUE TITLE
Cloud Jira integration not working with selfhosted sentry

### ISSUE THREAD

**Issue by sree-warrier** [getsentry/sentry#73354]
### Environment

self-hosted (https://develop.sentry.dev/self-hosted/)

### What are you trying to accomplish?

We were trying to integrate cloud jira with self hosted sentry but unable to achieve it. We are using sentry version 23.11.2

Following are the docs we followed
https://docs.sentry.io/organization/integrations/issue-tracking/jira/
https://forum.sentry.io/t/how-to-configure-jira-cloud-in-your-on-premise-sentry/6720

We have tried to use the app from jira market place but the app was redirecting to sentry cloud url (sentry.io). How we can configure the jira app so that it can redirect to self hosted senty url.

We have also tried app creation with `Atlassian Connect descriptor` https://develop.sentry.dev/integrations/jira/. Stuck on this part how we can integrate to selfhosted sentry

Let us know if any other docs are there

### How are you getting stuck?

Not able to integrate Jira cloud with self hosted sentry

### Where in the product are you?

Settings - Integrations

### Version

23.11.2

**Comment by markstory** [comment]
> We have also tried app creation with Atlassian Connect descriptor https://develop.sentry.dev/integrations/jira/. Stuck on this part how we can integrate to selfhosted sentry

What part of this are you getting stuck on? This is the flow we use for local development on the jira cloud integration as well.

**Comment by Cybernisk** [comment]
@sree-warrier I guess I know what the issue could be.
After you install the app in Jira and click setup, it will give you an error. Usually like 

![Image](https://github.com/user-attachments/assets/6b0f0535-3dd0-4be8-8af0-ee7627ed1eec)

There are several ways to solve this problem. Start by disabling AD and Identity blockers in your browser. But, in my experience, this will not help you. You need to open browser console (usually F12) and find the line like in the screenshot below and click the link. The URL should be your Sentry's domain. The issue is that Jira tab tries to open an iframe, which is normally blocked by security policies (X-Frame-Options) and you don't see a frame with a button to start the plugin setup. I hope this helps you to complete the configuration.
 

![Image](https://github.com/user-attachments/assets/d370b68a-7b1b-4b8f-840c-3f010833c07e)

**Comment by GabeVillalobos** [comment]
Unfortunately the team is understaffed and we can’t prioritize this fix but we’re always open to community contributions.

**Comment by mathroc** [comment]
was the issue closed on prupose?

**Comment by GabeVillalobos** [comment]
@mathroc It was. Our team has been closing stale issues for older releases. If this is still a problem on our latest release, please create a new issue so that our team can triage appropriately.
